frontend/src/app/users/users-profile/users-profile.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<h2>Profile of {{ userName }}</h2>
+
+<div *ngIf="userId !== currentUser?.id">
+  Common Projects
+
+  <ng-container
+    *ngIf="(projectService.projects$ | async)?.length; else elseBlock"
+  >
+    <mat-card
+      *ngFor="let project of projectService.projects$ | async"
+      class="!ml-0 cursor-pointer"
+      [routerLink]="['/project', project.slug]"
+    >
+      <mat-card-content>
```

Comment:
Common projects don't work if at least one admin is involved. For example, try to find common projects while logged in as admin. It always shows that there are no common projects.

Reply:
but what would you expect there to happen? I found this case but as admin I do not necessarily have any projects while I can still see all projects... does that mean I am in all projects and have common projects with every user?

Reply:
Consider an admin to have access to all projects.

User 1 has access to project 1: The admin would have the command project 1. 
User 2 has no access to any projects: There are no command projects with admins. 

More or less, the feature can be used for admins to see all projects the user has access to.